installscripts/cookbooks/Policyfile.rb

```diff
@@ -4,4 +4,4 @@
 cookbook 'nodejs', '~> 5.0.0'
 cookbook 'cloudcli', '~> 1.2.0'
 cookbook 'jenkins', path: './jenkins'
-run_list 'jenkins::setupjenkins', 'jenkins::sonarqube-scanner', 'jenkins::configurejenkins'
+run_list 'jenkins::setupjenkins', 'jenkins::sonarqube-scanner'
```

Comment:
You deleted `configurejenkins` from the Policyfile runlist but left it in the source tree. Either remove that recipe from the codebase entirely or use it.

If the latter, make sure you update the `script_root` chef variable to reflect your changes.

Reply:
Removed it.